services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -27,13 +30,22 @@ import {
 import { openDocument } from "../syncfusion/DocumentViewer";
 import { downloadFileFromDocumentManager } from "@common/utils/actionlessNetworkCalls";
 import { getUserAccessData } from "@common/selectors/authenticationSelectors";
+import { Dropdown, Button, MenuProps, notification, Progress } from "antd";
+import { DownOutlined } from "@ant-design/icons";
+import { USER_ROLES } from "@common/constants/environment";
```

Comment:
I think there's some unused imports in here.

Reply:
These have been removed now, thanks.